src/service/time-api-service/tests/test-time-api-service.js

```diff
@@ -42,6 +42,7 @@ describe('Timer API Service', function () {
     it('should respect server config', function () {
         var ts = new TimeService({
             server: {
+                protocol: 'https',
```

Comment:
why do the tests need this hardcoded? Shouldn't it default to https since you're running from localhost?

Reply:
@hkumar1993 bump. The tests should pass without having to hardcode http - if you think it's a bug in a 'expected' for a test, let me know.